server/@types/prisonerSearchApiClient/index.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module 'prisonRegisterApiClient' {
```

Comment:
This is the core problem, where we have incorrectly copy/pasted the module name from Prison Register API Client

The data types within this module are to do with prisoners, not prisons; and the data comes from the HMPPS Prisoner Search API.  So the module name should be `prisonerSearchApiClient`

The rest of the changes in this PR are simply correcting the imports based on the new module name